tests/end-to-end/grafana/authentication.cy.js

```diff
@@ -41,6 +41,13 @@ describe("grafana admin authentication", function() {
         }
       })
 
+    cy.yqDigParse("sc", ".grafana.ops.oidc.allowedDomains")
+      .then(domains => {
+        if (!domains.includes("example.com")) {
+          this.skip("example.com not set in .grafana.ops.oidc.allowedDomains")
+        }
+      })
```

Comment:
Should these checks also be in the integration tests?

Reply:
I don't see that we have integration tests for Grafana atm and adding new ones I feel like is out of scope for this PR

Reply:
Sorry I thought we had the same set, but that was for harbor. :sweat_smile: